api/src/db/model/quota.ts

```diff
@@ -81,6 +81,48 @@ export default class QuotaModel extends Model {
         // this is intentional (required by Sonarcloud)
     }
 
+    async findQuota(): Promise<any[]> {
```

Comment:
What's the difference you intend between method `findQuota` and base model method `findAll`?

Reply:
Nothing, it looks like I should have used findAll instead.
I've removed the findQuota function now.

Reply:
Nevermind - findAll() contains the query  `WHERE archived = false` and the db for the quota doesn't have a column for `archived`.
Because of this, don't we need the new `findQuota` function?